foodnet/membership/fixtures/initial_data.json

```diff
@@ -0,0 +1,27 @@
+[
+{
+    "model": "membership.membercategory",
+    "pk": 1,
+    "fields": {
+        "name": "houshold"
+    }
+},
+{
+    "model": "membership.membercategory",
+    "pk": 2,
+    "fields": {
+        "name": "school"
+    }
+},
+{
+    "model": "membership.division",
+    "pk": 1,
+    "fields": {
+        "allow_webmembers": true,
+        "category": "food",
+        "name": "default",
+        "contact": "admin@localhost",
+        "shortname": "def"
+    }
+}
+]
```

Comment:
mhm yes, so there is a user story for department types/categories and membership types/categories and I was thinking of doing this when implementing it. Also division should be renamed to department.